### PR TITLE
[doc][feat] Added doc to deploy a pulsar cluster on IBM cloud services

### PR DESCRIPTION
**Pull request by 5exceptions**
### Modifications
Added docs to deploy a pulsar cluster on **IBM cloud services**.  

Alt text

### Documentation
- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@tjiuming Could you please review this PR from a technical perspective? Then writers will review it from a technical writing side. Thank you!

**Comment by Anonymitaet** [comment]
ping @tjiuming to review

**Comment by tjiuming** [comment]
LGTM, but I'm not particularly familiar with  helm chart. Could @mattisonchao help review?

**Comment by Anonymitaet** [comment]
@5exceptions To verify everything goes smoothly, can you attach all preview screenshots of docs like what we do in https://github.com/apache/pulsar-site/pull/403#issuecomment-1423546100?

**Review by mattisonchao — APPROVED** [review]
LGTM, Assume the author has verified all the processes.

@5exceptions  Thanks for your nice contribution.

**Comment by 5exceptions** [comment]
Hi @mattisonchao 

Yes, we have verified all the process and you can find the screenshot in the document itself.
Thank you very much for approving our request.

**Comment by Anonymitaet** [comment]
> @5exceptions To verify everything goes smoothly, can you attach all preview screenshots of docs like what we do in [#403 (comment)](https://github.com/apache/pulsar-site/pull/403#issuecomment-1423546100)?

Hi @5exceptions any feedback on this? Thank you!

**Comment by 5exceptions** [comment]
Hi @Anonymitaet 

Please find and review few screenshots below.

k8s3

k8s10

Secretsout

PodStatus

VM11

Please let me know if any other detail is required 

Thanks

**Comment by Anonymitaet** [comment]
I mean the previews of markdown content. 

image

Can you take a closer look at these examples?
> @5exceptions To verify everything goes smoothly, can you attach all preview screenshots of docs like what we do in [#403 (comment)](https://github.com/apache/pulsar-site/pull/403#issuecomment-1423546100)?

File: versioned_docs/version-2.10.x/deploy-ibm.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+ kubectl get services
+``` 
+
+Output
+![Output of Command Services](/assets/IBMCloud/Services.png)
+
+> Output of above command will give us `services URL` and `broker URL`, proxy external ip is for both just change ports (for service URL port is 80 and for broker URL port is 6650 this is default)
+If all of these commands are successful, Pulsar clients can now use your cluster!
+
+> You can use Pulsar client to publishing and consuming messages you just need to use `proxy external IP` and ports of `broker URL` and `service URL`.
```
[the remark this answers is not in the document]
Comment (Anonymitaet) [reply versioned_docs/version-2.10.x/deploy-ibm.md:237]: Suggest adding a simple example of producing and consuming messages here. In this way, it's a complete tutorial.

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution!  I've left some comments, PTAL. 

You can update 2.10.x first and then update other versioned docs once we reach a consensus on the changes. 

Feel free to correct me if my understanding is inaccurate and let me know if you have any concerns. 😊

Feel free to leave your suggestions on the writing. @momo-jun @DaveDuggins @D-2-Ed

**Comment by 5exceptions** [comment]
Hi @Anonymitaet 

Thank you for reviewing the document and providing your feedback.
As per the review's comment, we have made appropriate changes and committed the doc. 
Please review it again.

**Comment by Anonymitaet** [comment]
Hi @5exceptions I see some of my comments have been ignored and no reasons were given. Any thoughts?

I just comment on some examples, can you check overall one by one?

**Comment by 5exceptions** [comment]
Hi @Anonymitaet 

Sorry, in last commit, I missed some of the feedback.
I have fixed them now. Please review it again and provide your feedback.
Thanks

**Review by Anonymitaet — APPROVED** [review]

**Comment by 5exceptions** [comment]
Thank you very much for approving and merging the request.

**Comment by leaveamemo** [comment]
Thank you @5exceptions, @Anonymitaet, and @mattisonchao for the great collaboration on this!!!

**Comment by Anonymitaet** [comment]
@5exceptions I've checked the website and found some issues, can you help revolve? Thank you!

1. This doc is not added to `master`. If this content suitable for `master`, can you add it?

3. For 2.11.x, the doc is not correctly shown.
image

4. For 2.10.x and 2.9.x, the right navigation is not correctly shown. You might need to update headings.
image

**Pull request merged**